deadline_io.go

```diff
@@ -0,0 +1,31 @@
+package tlsdialer
+
+import (
+	"net"
+	"time"
+)
+
+type deadlineIO struct {
+	to time.Duration
+	c  net.Conn
+}
+
+func (d *deadlineIO) Write(b []byte) (n int, err error) {
```

Comment:
Let's also add a unit-test to ensure that `diadlineIO` works fine. So we need to test:

https://github.com/tarantool/go-tlsdialer/blob/927312d079c272de904d378c9ab9793d329757a4/dialer.go#L24

that `tarantool.DialOpts`value applies successfully and work as expected.

Reply:
Added new test to the `integration_test.go`.